chapters/sampling-randomization-power.tex

```diff
@@ -1,391 +1,433 @@
-%------------------------------------------------
+%-----------------------------------------------------------------------------------------------
 
 \begin{fullwidth}
-Sampling, randomization, and power calculations are the core elements of experimental design.
-\textbf{Sampling} and \textbf{randomization} determine
-which units are observed and in which states.
-Each of these processes introduces statistical noise
+Sampling and randomization are two core elements of study design.
```

Comment:
Can we change _study design_ to _research design_ so it ties better into the previous chapter where we talked about _research design_ and never _study design_.

Reply:
882532318a6b7082310db847c304b14db94c6f9e in all locations in chapter